src/fileUtils.ts

```diff
@@ -0,0 +1,15 @@
+import * as vscode from 'vscode';
+
+export async function getFilePath(): Promise<vscode.Uri | undefined> {
+    const fileUri = await vscode.window.showSaveDialog({
+        defaultUri: vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders[0].uri : undefined,
+        filters: {
+            'xml': ['xml'],
+        }
+    });
+
+    if (!fileUri) {
```

Comment:
IIUC when `showSaveDialog` returns `undefined` that means the user pressed cancel, so I don't think a warning message should be displayed in that case. @QuocTrung76   Have a quick test that when you press cancel that no warning message is shown

Reply:
thank you, I removed the warning.

Reply:
confirmed 👍